overlord/snapstate/snapstate.go

```diff
@@ -450,11 +464,8 @@ func revertToRevision(s *state.State, name string, rev snap.Revision) (*state.Ta
 	if i < 0 {
 		return nil, fmt.Errorf("cannot find revision %s for snap %q", rev, name)
 	}
-	revertToRev := snapst.Sequence[i].Revision
-
 	ss := &SnapSetup{
-		Name:     name,
-		Revision: revertToRev,
+		SideInfo: snapst.Sequence[i],
```

Comment:
@mvo5 sorry, nothing from revert was in 2.0.10 yet right? so this is not a problem actually